src/igvfd/tests/test_upgrade_open_reading_frame.py

```diff
@@ -1,6 +1,16 @@
 import pytest
 
 
+def test_open_reading_frame_upgrade_2_3(upgrader, open_reading_frame_v2):
+    """Test upgrade from ORF Schema 2 to Schema 3"""
+    value = upgrader.upgrade('open_reading_frame', open_reading_frame_v2, current_version='2', target_version='3')
+    assert value['notes'] == 'This object used gene as a property, which is now upgraded to genes.'
+    assert value['genes'] == [
+        '/genes/ENSMUSG00000023902/',
+    ]
+    assert value['schema_version'] == '3'
+
+
```

Comment:
Later upgrade tests should go below earlier upgrade tests.

Reply:
Moved.